backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3995,5 +3995,32 @@ databaseChangeLog:
               FROM ${database.defaultSchemaName}.test_order t
               LEFT JOIN ${database.defaultSchemaName}.result ON t.internal_id=${database.defaultSchemaName}.result.test_order_id 
               WHERE ${database.defaultSchemaName}.result.test_order_id IS NULL AND t.result IS NOT NULL;
+      rollback:
+        sql: SELECT 'foo';
+  - changeSet:
+      id: backfill-results-table-with-removals-and-corrections
+      author: emma@skylight.digital
+      comment: backfill the new results table with TestEvents that have been removed or corrected
+      changes:
+        - tagDatabase:
+            tag: backfill-results-table-with-removals-and-corrections
+        - sql:
+            sql: |
+              INSERT INTO ${database.defaultSchemaName}.result
```

Comment:
(non-blocking) where is the `database.defaultSchemaName` being set? 🤔

Reply:
Good question! I believe it's being set as part of our initial database connection. The variable is defined in the build.gradle file, since it varies per environment: https://github.com/CDCgov/prime-simplereport/blob/002c972e4649fb4f28b35e8363bebe20fee58750/backend/build.gradle#L160-L160

And then passed to liquibase as a parameter:
https://github.com/CDCgov/prime-simplereport/blob/002c972e4649fb4f28b35e8363bebe20fee58750/backend/build.gradle#L171-L171